coffeecard/CoffeeCard.Library/Services/AccountService.cs

```diff
@@ -94,31 +94,29 @@ public string Login(string email, string password, string version)
                 StatusCodes.Status401Unauthorized);
         }
 
-        public async Task<User> RegisterAccountAsync(RegisterDto registerDto)
+        public async Task<User> RegisterAccountAsync(string name, string email, string password, int programme = 1)
```

Comment:
Document what programme = 1 means, e.g. SWU. However, does it make sense to have a default value, when is that the case?

Reply:
Ahh, I suppose for backwards compatibility?

Reply:
Yeah, its for backwards compatibility, so the call in the other controller does not need to add it.